packages/client/src/runtime/getPrismaClient.ts

```diff
@@ -1077,12 +1075,12 @@ new PrismaClient({
           // if this `next` was called and there's some more middlewares
           const nextMiddleware = this._middlewares.query.get(++index)
 
-          if (nextMiddleware) {
-            // we pass the modfied params down to the next one, & repeat
-            return nextMiddleware(changedParams, consumer)
-          }
+          // we pass the modified params down to the next one, & repeat
+          if (nextMiddleware) return nextMiddleware(changedParams, consumer)
+          // calling `next` calls the consumer again with the new params
```

Comment:
does this comment apply to the line before?

Reply:
Yes

Reply:
I would probably not understand that when reading the code - but maybe that is just me. Feel free to ignore and resolve comment.